acceptance/tests/base/host/packages.rb

```diff
@@ -3,23 +3,23 @@
 
 def get_host_pkg(host)
   case
-    when /sles-10/.match?(host['platform'])
+    when host['platform'].include?('sles-10')
```

Comment:
I wonder if this whole case statement should be rewritten to a `case host['platform']` and then use matchers in `when`.

Reply:
So what I meant (and may be a smaller diff)
```ruby
def get_host_pkg(host)
  case host['platform']
  when /sles-10/
    Beaker::HostPrebuiltSteps::SLES10_PACKAGES
  when /opensuse|sles-/
    Beaker::HostPrebuiltSteps::SLES_PACKAGES
  when /debian/
    Beaker::HostPrebuiltSteps::DEBIAN_PACKAGES
  when /cumulus/
    Beaker::HostPrebuiltSteps::CUMULUS_PACKAGES
  when /windows/
    host.is_cygwin? ? Beaker::HostPrebuiltSteps::WINDOWS_PACKAGES : Beaker::HostPrebuiltSteps::PSWINDOWS_PACKAGES
  when /freebsd/
    Beaker::HostPrebuiltSteps::FREEBSD_PACKAGES
  when /openbsd/
    Beaker::HostPrebuiltSteps::OPENBSD_PACKAGES
  when /solaris-10/
    Beaker::HostPrebuiltSteps::SOLARIS10_PACKAGES
  else
    Beaker::HostPrebuiltSteps::UNIX_PACKAGES
  end
end
```

but I'm fine with this too